uploader.json

```diff
@@ -793,6 +793,11 @@
             "pattern": [],
             "symbolic_name": "ntptime"
         },
+        {
+            "command": "/usr/sbin/fcoeadm -i",
+            "pattern": [],
+            "symbolic_name": "fcoeadm_i"
```

Comment:
@xhe123 we try to keep these in alphanumeric order based on the symbolic name.  Please move this change to after the symbolic name `fc_match`.